starlette/applications.py

```diff
@@ -115,13 +115,14 @@ def debug(self) -> bool:
     @debug.setter
     def debug(self, value: bool) -> None:
         self._debug = value
```

Comment:
We can now replace this setter by just using `self.debug`, can't we?

Reply:
I think this PR is now _less_ lines of code and less complexity 🥳